lib/src/ffmpeg/filters/afade_curve.dart

```diff
@@ -0,0 +1,43 @@
+/// Options include
```

Comment:
If these values go to a filter, then shouldn't those filters be in this file? Or shouldn't this content be in the filter's file?

Reply:
I now put the enum inside of the fade_filter.dart file as I see what you are saying. 
Although, I want to point out that there is a discrepancy with the sws.dart file for the SwsFlag class which is only used inside of the scaleFilter but it has its own file.
This is partly the reason why I created a new file.

Reply:
Ok, it's possible that sws.dart was an oversight on my part. I've been working with incomplete knowledge of FFMPEG this whole time, so there might be any number of places where I did the wrong thing. In general, if a data structure or enum only applies to behavior within a single file, that's usually where we should put the data structure or enum.